### PR TITLE
FK constraint for .texts entities

### PR DESCRIPTION
**Pull request by stewsk**

### CODE REVIEW

File: guides/databases.md [hunk 1]
```diff
@@ -642,9 +644,8 @@ No constraints are generated for...
 * Associations annotated with `@assert.integrity: false`
 * Associations which's source or target entity is annotated with `@cds.persistence.exists` or `@cds.persistence.skip`
 
-
-
-If the association is the backlink of a **composition**, the constraint's delete rule changes to `CASCADE`. For example that applies to the `parent` association in here:
+If the association is the backlink of a **composition**, the constraint's delete rule changes to `CASCADE`.
+For example that applies to the `parent` association in here:
```
Comment (renejeglinsky) [inline guides/databases.md:648]:
```suggestion
That applies, for example, to the `parent` association in here:
```

File: guides/databases.md [hunk 2]
```diff
@@ -654,7 +655,35 @@ entity Genres {
 }
 ```
 
+As a special case, a referential constraint with `delete cascade` is also generated
+for text table of a [localized entity](../guides/localized-data#localized-data),
```
Comment (renejeglinsky) [inline guides/databases.md:659]:
```suggestion
for the text table of a [localized entity](../guides/localized-data#localized-data),
```

File: guides/databases.md [hunk 3]
```diff
@@ -654,7 +655,35 @@ entity Genres {
...
+although no managed association is present in the `texts` entity.
+
+Add a localized element to entity `Books` in the example above:
```
Comment (renejeglinsky) [inline guides/databases.md:662]: Does the reference to the previous example really matter here? Seems self-contained to me.
```suggestion
Add a localized element to entity `Books` from the previous example:
```

```suggestion
Add a localized element to entity `Books`:
```

File: guides/databases.md [hunk 4]
```diff
@@ -654,7 +655,35 @@ entity Genres {
 }
 ```
 
+As a special case, a referential constraint with `delete cascade` is also generated
+for the text table of a [localized entity](../guides/localized-data#localized-data),
+although no managed association is present in the `texts` entity.
+
+Add a localized element to entity `Books`:
```
Comment (stewsk) [inline guides/databases.md:662]:
```suggestion
Add a localized element to entity `Books` from the previous example:
```

**Comment by stewsk** [comment]
> Does the reference to the previous example really matter here? Seems self-contained to me.

Usually the Books entity always has a localized element. So here I think it makes sense to refer to the example above where we first have defined Books w/o localized element.

**Comment by stewsk** [comment]
ok for me

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**